packages/client/src/NextNProgress.tsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 
 import { css, Global } from '@emotion/react';
-import RouterImport from 'next/router';
+import RouterImport from 'next/router.js';
```

Comment:
why it's necessary? and other cases with `index.js`

Reply:
this is part of esm, it's not new

Reply:
Next.js doesn't have explicit package exports, which means that deep imports need explicit extensions, it's just how it works